apps/meteor/ee/app/livechat-enterprise/server/api/tags.ts

```diff
@@ -27,7 +27,7 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'livechat/tags/:tagId',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: ['view-l-room'] },
```

Comment:
Missing permission: manage-livechat-tags

Reply:
Also, since we have a permission check here, so can we remove it from `findTagById` to avoid re-checking?